plotting_functions.py

```diff
@@ -0,0 +1,1653 @@
+"""
+This file contains functions to generate figures of the logic tree investigation results.
+"""
+
+from pathlib import Path
+from typing import Union
+
+import natsort
+import numpy as np
+import pyarrow.dataset as ds
+import toml
+import toshi_hazard_post.calculators as calculators
+from matplotlib import pyplot as plt
+
+import plotting_helpers
+
+
+def make_figure_of_coefficient_of_variation(
+    results_directory: Union[Path, str],
+    plot_output_directory: Union[Path, str],
+    plot_dpi: int = 500,
+    plot_fontsize: float = 12.0,
+    plot_lineweight: float = 5.0,
+    location: str = "WLG",
+    im: str = "PGA",
```

Comment:
A [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) for this would be preferable to a raw string.

Reply:
Changed string arguments like these to StrEnum